controllers/topology_controller.go

```diff
@@ -156,7 +156,17 @@ func (r *TopologyReconciler) handleRMQReferenceParseError(ctx context.Context, o
 		}
 		return ctrl.Result{}, nil
 	}
+
+	// set status condition and publish event for any other error
 	logger.Error(err, failedParseClusterRef)
+	msg := fmt.Sprintf("%s: %s", failedParseClusterRef, err.Error())
+	r.Recorder.Event(object, corev1.EventTypeWarning, "FailedCreateOrUpdate", msg)
```

Comment:
Just FTR, I wanted to suggest using a "type error" here but apparently there only "normal" and "warning" to choose from in the API :)